src/main.test.ts

```diff
@@ -0,0 +1,102 @@
+import * as prData from '../fixtures/getAllWorkResponse.fixture.json';
```

Comment:
The idea of overarching tests is great. I would expect, however, that the `draft_pr` tests would be within an `openPR.test.ts` file since that's where the different variable would show up.

Reply:
I didn't see any other examples in this repo of actually testing the input that gets passed into the action. I felt like just testing that the openPR function passed whatever it argument received for `draft_pr` to the API call wouldn't be that helpful. 

I wanted to test that the API call gets called with the correct arguments based on various the input from the actual action.

Since I didn't see anything like that in this repo (like, there's no tests for passing in different values for the `timespan` input, for example), I referenced other actions like actions/labeler, which had tests at the main.test.ts level for various inputs to the action: https://github.com/actions/labeler/blob/main/__tests__/main.test.ts#L66

Reply:
I could move this all into `openPR.test.ts` if you'd prefer! I was thinking that this file would be for all action-level end-to-end type tests, and something like `openPR.test.ts` would be more like just testing that function

Reply:
@octokatherine - I really dropped the ball here as this got lost in my to-dos. Yes please move the `draft_pr` tests into `openPR.test` 🙏